inspirehep/utils/bibtex.py

```diff
@@ -318,7 +318,7 @@ def _get_collaboration(self):
         collaboration = ''
         if 'collaboration' in self.record:
             try:
-                collaboration = self.record['collaboration'][0]
+                collaboration = self.record['collaboration'][0]['value']
             except IndexError:
```

Comment:
May also need a `KeyError` catch here after `value` was added. If `self.record` is a `Record` object we can in fact use ```self.record['collaboration.value']```